hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/common/model/HoodieSparkRecord.java

```diff
@@ -76,6 +80,8 @@
  *
  */
 public class HoodieSparkRecord extends HoodieRecord<InternalRow> {
+  private static final Map<Pair<StructType, Schema>, Function<InternalRow, GenericRecord>>
+      AVRO_CONVERTER_CACHE = new ConcurrentHashMap<>();
```

Comment:
should we just move the cache into `AvroConversionUtils`?

Reply:
Yes, will move this